util/src/main/java/io/grpc/util/AdvancedTlsX509TrustManager.java

```diff
@@ -356,6 +379,19 @@ void verifyPeerCertificate(X509Certificate[] peerCertChain, String authType, SSL
         throws CertificateException;
   }
 
+  /** Builds a new {@link AdvancedTlsX509TrustManager}.
+   * By default, no trust certificates are loaded after the build. To load them, use one of the
+   * following methods:
+   * {@link AdvancedTlsX509TrustManager#updateTrustCredentials(X509Certificate[])},
+   * {@link AdvancedTlsX509TrustManager#updateTrustCredentialsFromFile
+   * (File, long, TimeUnit, ScheduledExecutorService)},
+   * {@link AdvancedTlsX509TrustManager#updateTrustCredentialsFromFile
+   * (File, long, TimeUnit, ScheduledExecutorService)}
+   * By default, {@link Verification#CERTIFICATE_AND_HOST_NAME_VERIFICATION} mode for authenticating
```

Comment:
This "newline before each sentence" is not great. Flow it together, or separate it more and put `<p>`. I can understand it sometimes because you don't want to reflow the text, but here it is every sentence and that is quite different from what it will look like in the generated doc. The sentence just before this doesn't have a period, which would be more obvious if it was flowed together.

Reply:
Ok I played a little with reflowing, this is how javadoc looks now:
![Image 6-7-24 at 9 35 PM](https://github.com/grpc/grpc-java/assets/110920239/1c299508-cbed-4bcf-aee7-4a4e4b45a073)